features/consent/src/main/java/in/projecteka/jataayu/consent/ui/fragment/RequestedListFragment.kt

```diff
@@ -196,4 +184,13 @@ class RequestedFragment : BaseFragment(), AdapterView.OnItemSelectedListener, It
     fun getConsentFlow(): ConsentFlow {
         return ConsentFlow.REQUESTED_CONSENTS
     }
+
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        super.onActivityResult(requestCode, resultCode, data)
```

Comment:
as discussed, use the data: Intent? to clearly dictate what needs to be done. 
Also, use a separate event to trigger refresh. Pull to refresh event could be doing other things.